### DISCUSSION TITLE
Galaxy Badge

### DISCUSSION THREAD

**Discussion by Saqib198** [community/community#125653] [Other Feature Feedback, Questions, & Ideas]
To unlock the Galaxy Brain badge on GitHub, you need to answer at least 2 discussions. The discussions must be in a public repository, and you cannot mark your own answers to your own questions.

**Comment by matteofuso** [comment]
Exactly, to get the Galaxy Brain Badge you must

1. Get at least 2 marked answers in public discussion
2. You cann't mark your own responses

You may need more than 2 responses to discussions

**Comment by MarkCyber** [comment]
This is correct. I got my first answered question yesterday and was surprised I didn't get it. But the answer needs to get accepted as the best answer, and it also needs to happen twice on two different discussions. Good luck! Hopefully I'll get it too ;)

**Comment by CopperEagle** [comment]
Hey @Saqib198 

Please note that this must happen in another repositories' discussion section. In the community section, this was disabled after cheating occured. See announcement [here](https://github.com/orgs/community/discussions/106536)
